Common/Data/Custom/Fred/Wilshire.cs

```diff
@@ -0,0 +1,351 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+namespace QuantConnect.Data.Custom.Fred
+{
+    public static partial class FredSeries
+    {
+        /// <summary>
+        /// Wilshire Indexes help clients, investment professionals and researchers accurately measure and better understand the market. The Wilshire Index family leverages more than 40 years of Wilshire performance measurement expertise and employs unbiased construction rules.
+        /// </summary>
+        public static class Wilshire
+        {
+            ///<summary>
+            /// Wilshire US Small-Cap Value Price Index (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/WILLSMLCAPVALPR
+            /// </summary>
+            /// <remarks>
+            /// The price indexes are price returns, which do not reinvest dividends. Copyright, 2016, Wilshire Associates Incorporated. Reprinted with permission. More information about the various indexes from Wilshire Associates can be found at http://www.wilshire.com/Indexes/.
+            /// </remarks>
+            public static string WilshireUSSmallCapValuePrice => "WILLSMLCAPVALPR";
```

Comment:
Redundant `Wilshire` in all variable names even though the class is named `Wilshire`. We'd be accessing it via `FredSeries.Wilshire.Wilshire...`

Reply:
I left `Wilshire` in this case because many of the properties start with a number, e.g.  `Willshire2500GrowthPrice` even when it is redundant, I consider it is better than `Willshire.TwoThousandFiveHundredGrowthPrice`.